main/meleeMenu.h

```diff
@@ -12,7 +12,8 @@
 // Defines
 //==============================================================================
 
-#define MAX_ROWS 6
+#define MAX_ROWS 256
+#define MAX_ROWS_ON_SCREEN 6
```

Comment:
If you allow scrolling, this should be 5 like the OG menu.
![image](https://user-images.githubusercontent.com/231180/205504934-b9c2e8e5-7c26-4e6a-be03-52fa06e74810.png)

Reply:
I didn't want to mess up menus that had planned for 6 being the default, and I also didn't want to re-plan the spacing, but I can adjust if you prefer 5

Reply:
If you look far back enough, it was written for 5, and got bumped to 6 when something got big. I don't love how the last item overlaps the colored border, but 6 was easier for my sanity than whatever the alternative was at the time.

Reply:
Oh, I didn't even notice that it wasn't supposed to overlap the border; thought that was some sort of asymmetrical design like the different row offsets.

So that's a "yes, switch to 5 rows", right?

Reply:
I assumed that was a yes, and implemented it